Functions.Tests/GeoTrack.Tests.ps1

```diff
@@ -0,0 +1,26 @@
+$here = $PSScriptRoot
+$sut = $PSCommandPath.Replace('.Tests', '')
+. $sut
+
+
+Describe "Get-GoogleSessionVariable" {
+    It "Gets a WebRequestSession with Google session cookies" {
+        $session = Get-GoogleSessionVariable intellitectpowershelltests@gmail.com pstestaccount
```

Comment:
Unfortunately, this is going to prevent us ever pushing this code to OSS.  Hmm... perhaps use CredentialManagerCredential so that we pull from credentials a local repository.  And, if they aren't there issue a warning WITH INSTRUCTIONS on how to fix, but discontinue the test with an inconclusive state (does Pester support this?)